<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="photo_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="setpass_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>setpass_btn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restart_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>restart_btn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ok_btn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetpass_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>resetpass_btn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetbg_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>resetbg_btn</string>
   </property>
   <property name="icon">
    <iconset resource="res/res.qrc">
     <normaloff>:/res/reset.bmp</normaloff>:/res/reset.bmp</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="warning_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>warning_label</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
